src/scores/continuous/consistent_impl.py

```diff
@@ -0,0 +1,262 @@
+"""
+Implementation of scoring functions that are consistent for
+single-valued forecasts targeting quantiles, expectiles or Huber functionals.
+"""
+
+from typing import Callable, Optional
+
+import xarray as xr
+
+from scores.functions import apply_weights
+from scores.typing import FlexibleDimensionTypes
+from scores.utils import gather_dimensions
+
+
+def consistent_expectile_score(
+    fcst: xr.DataArray,
+    obs: xr.DataArray,
+    *,  # Force keywords arguments to be keyword-only
+    alpha: float,
```

Comment:
FYI, putting the "*" before alpha here forces alpha to be a keyword argument. If you put the "*" after the alpha, then alpha may be supplied as a positional argument. Since alpha is a mandatory argument (i.e. no default is supplied), this means alpha is always required and always a keyword. This is totally fine if that's what you want, but it's not necessarily what you were aiming for.

To force only optional arguments to be keyword arguments, put the "*" between phi_prime and reduce_dims.

That said, I have almost reached the point where I consider positional arguments inherently mistake-prone and wonder if the "*" should just go first all the time... :/ .

Feel free to leave the "*" where it is or move it between the positional and optional arguments as you see fit. Leaving it where it is, is completely fine.

Reply:
I'll shift the "*" further down as this will be more consistent with the rest of the repository.